content/en/docs/concepts/architecture/cri.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Container Runtime Interface (CRI)
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+You need a working
+{{<glossary_tooltip text="container runtime" term_id="container-runtime">}} on
+each Node in your cluster, so that the
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} can launch
+{{< glossary_tooltip text="Pods" term_id="pod" >}} and their containers.
+
+{{< glossary_definition term_id="container-runtime-interface" length="all" >}}
+
+The CRI is a plugin interface which enables the kubelet to use a wide variety of
+container runtimes, without having a need to recompile the cluster components.
+
+<!-- body -->
+
+## The API {#api}
+
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
+
+The kubelet acts as a client when connecting to the container runtime via gRPC.
+For Kubernetes v{{< skew currentVersion >}}, the kubelet prefers to use CRI `v1`.
+If a container runtime does not support `v1` of the CRI, then the kubelet tries to
+negotiate any older supported version.
+The v{{< skew currentVersion >}} kubelet can also negotiate CRI `v1alpha2`.
```

Comment:
Maybe mention that this is the behavior for now and is subject for deprecation.

Reply:
We typically [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future) unless we're committing to them.

Reply:
ok, it is already printing that v1alpha2 is deprecated. So worth mentioning it here.

This is what printed on fallback:

```
remote_runtime.go:118] "Falling back to CRI v1alpha2 runtime API (deprecated)"
remote_image.go:94] "Falling back to CRI v1alpha2 image API (deprecated)"
```

Reply:
twice because of image services (other comment)

Reply:
Added a note that it's now deprecated :+1: